test/scripts/deploy-gcp-bench-cluster.sh

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+CLUSTER_NAME=contour-bench
```

Comment:
would it make sense to add a random component to the name to avoid conflicts?

Reply:
yep, can add that!

was going to have this be used in a "nightly" CI run since the full tests are relatively expensive and takes a while so *hopefully* collisions won't be an issue